backend/src/errors/index.ts

```diff
@@ -34,6 +34,10 @@ export function clientUnknown(): Response {
 	return generateErrorResponse(`The client is unknown or invalid`, 403)
 }
 
+export function notFound(): Response {
```

Comment:
Could you please remove the 404 change from this PR? I think it needs a change

Reply:
Yep sorry that was not intentional.